### ISSUE TITLE
Import of setup duplicated in setup.py

### ISSUE THREAD

**Issue by hadfieldnz** [powellb/seapy#61]
In setup.py on the master branch at lines 6 & 7, the setup function is imported twice.
```
from setuptools import find_packages, setup
from numpy.distutils.core import setup
```
The effect of the second import is to replace the first. This gives the correct behaviour (as numpy.distutils.core is the one required) but is potentially confusing. I recommend
```
from setuptools import find_packages
from numpy.distutils.core import setup
```
Priority is low as there is no change in function. I have not prepared a PR.

**Comment by powellb** [comment]
Thanks, I'll fix when I merge in the latest updates from the development branch.

**Comment by powellb** [comment]
Sorry for the delay. This has been fixed and new version is tagged as 0.5.0.

**Comment by hadfieldnz** [comment]
Thanks.

Mark Hadfield
51 Volga St
Wellington 6023, NZ
